content/en/docs/contribute/generate-ref-docs/kubernetes-components.md

```diff
@@ -118,134 +118,79 @@ need to work with someone who can set the label and milestone for you.
 
 The `update-imported-docs` tool performs these steps:
 
-1. Clone the `kubernetes/kubernetes` repository.
-1. Run several scripts under `kubernetes/kubernetes/hack`. These scripts
-   generate Markdown files and place the files under `kubernetes/kubernetes/docs`.
+1. Clone the related repositories specified in a configuration file. For the
+   purpose of generating reference docs, the repositories that are cloned by
+   default are `kubernetes-incubator/reference-docs` and `kubernetes/federation`.
+1. Run commands under the cloned repositories to prepare the docs generator and
+   then generates the Markdown doc files.
 1. Copy the generated Markdown files to a local clone of the `kubernetes/website`
-   repository under `kubernetes/website/docs/reference/generated`.
-1. Clone the `kubernetes/federation` repository.
-1. Run several scripts under `kubernetes/federation/hack`. These scripts
-   generate Markdown files and place the files under `kubernetes/federation/docs`.
-1. Copy the generated Markdown files to a local clone of the `kubernetes/website`
-   repository under `kubernetes/website/docs/reference/generated`.
+   repository under locations specified in the configuration file.
 
-After the Markdown files are in your local clone of the `kubernetes/website`
+When the Markdown files are in your local clone of the `kubernetes/website`
 repository, you can submit them in a
 [pull request](https://kubernetes.io/docs/home/contribute/create-pull-request/)
 to `kubernetes/website`.
 
-## Setting the branch
-
-Open `<web-base>/update-imported-docs/config.yaml` for editing.
+## Customizing the config file
 
-Set the value of `branch` to the Kubernetes release that you want to document.
-For example, if you want to generate docs for the Kubernetes 1.9 release,
-set `branch` to `release-1.9`.
+Open `<web-base>/update-imported-docs/reference.yaml` for editing.
+Do not change the content for the `generate-command` entry unless you know what
+you are doing.
 
 ```shell
 repos:
-- name: kubernetes
-  remote: https://github.com/kubernetes/kubernetes.git
-  branch: release-1.9
+- name: reference-docs
+  remote: https://github.com/kubernetes-incubator/reference-docs.git
+  # This and the generate-command below needs a change when reference-docs has
+  # branches properly defined
+  branch: master  
+  generate-command: |
+    cd $GOPATH
+    git clone https://github.com/kubernetes/kubernetes.git src/k8s.io/kubernetes
+    cd src/k8s.io/kubernetes
+    git checkout release-1.11
```

Comment:
Do we want to make release-1.11 a variable?

Reply:
Hi, @tengqm. Sorry, I am coming back to the generate-command. I am still trying to figure out the process.
- Specifying an external script for generate-command seems useful, especially if this script can be used in multiple places in reference.yaml.
- Will the federation component defined in reference.yaml use the generate-command?

Reply:
@kbhawkey Not sure how triggering works. Some examples would be helpful for feasibility evaluation.